openrtb_ext/imp_adhese.go

```diff
@@ -1,12 +1,8 @@
 package openrtb_ext
 
-import (
-	"encoding/json"
-)
-
 type ExtImpAdhese struct {
-	Account  string          `json:"account"`
-	Location string          `json:"location"`
-	Format   string          `json:"format"`
-	Keywords json.RawMessage `json:"targets,omitempty"`
+	Account  string              `json:"account"`
+	Location string              `json:"location"`
+	Format   string              `json:"format"`
+	Targets  map[string][]string `json:"targets,omitempty"`
```

Comment:
@mefjush should update bidder docs to add description for `Targets` ext param

Reply:
The field is called `data` for pbjs, but it got renamed to `targets` in both the server-side adapters. I've added a column in the documentation where we specify the pbs ext name next to the pbjs configuration param name.
https://github.com/prebid/prebid.github.io/pull/5618/files